src/components/Nav/UserFormComponent/UserFormInputComponent/UserFormInputs.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import {
+  UserFromInputContainer,
+  InputBox,
+  InputTextBox,
+  InputText,
+  Input,
+} from './UserFormInputsEle';
+
+function UserFromInputs({ form }) {
+  return (
+    <UserFromInputContainer>
```

Comment:
```suggestion
    <Container>
```
저는 이 친구도 그냥 container로!